### PR TITLE
[CSL-1964] Document swagger API testing

### PR DESCRIPTION
**Pull request by adinapoli-iohk**
@akegalj Let me know how you get along with the instructions!

### CODE REVIEW

File: wallet-new/README.md [hunk 1]
```diff
@@ -1,26 +1,49 @@
 # Cardano Wallet API
 
-May this be the beginning of a beautiful program.
```
Comment (akegalj) [inline wallet-new/README.md]: :D

File: wallet-new/README.md [hunk 2]
```diff
@@ -1,26 +1,49 @@
...
+Running the command above *from the root of the Cardano project* will store an updated `swagger.json` into
+`wallet-new/spec`.
 
-We export the full Swagger specification inside `spec/swagger.json`.
+### Playing with the API
+
+Once you have your updated `swagger.json` file, the easiest way is to head over to [the online editor](https://editor.swagger.io),
+click "Edit -> Import File" and import the file. Once done that, the API will be rendered in its full glory.
+
+Alternatively (and *recommended*), is also possible to download the editor locally and play with it. This has the advantage
```
Comment (akegalj) [inline wallet-new/README.md]: Tested this method. Words perfectly.

File: wallet-new/README.md (hunk 1 above)
Comment (adinapoli-iohk) [reply wallet-new/README.md]: For the nostalgic, this was the sentence that [SLIME](https://github.com/slime/slime) showed upon opening the REPL :)

File: wallet-new/README.md [hunk 3]
```diff
@@ -1,26 +1,49 @@
...
+you can now also *try out* the API, because the `host` of the swagger API points to `localhost`, which won't work, of course,
+in case of the online editor. We won't get too deep into how to setup the editor locally, but generally speaking it should be
+as simple as:
+
+- Downloading [the editor](https://github.com/swagger-api/swagger-editor/archive/v3.1.17.zip) online;
```
Comment (Martoon-00) [inline wallet-new/README.md]: By the way, this editor can be launched simply by browser, e.g. `firefox swagger-editor/index.html`. Perhaps it's worth mentioning

Comment (akegalj) [reply wallet-new/README.md]: true, that's useful

**Pull request merged**